ae_network_connection_exception.gemspec

```diff
@@ -1,21 +1,26 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'ae_network_connection_exception/version'
+# frozen_string_literal: true
+
+require_relative 'lib/ae_network_connection_exception/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "ae_network_connection_exception"
+  spec.name          = 'ae_network_connection_exception'
   spec.version       = AeNetworkConnectionException::VERSION
-  spec.authors       = ["Steven Boyd"]
-  spec.email         = ["engineering@appfolio.com"]
-  spec.summary       = "Provides sane exceptions for network failures"
-  spec.description   = "A simple gem that will catch all manner of exceptions related to establishing a network connection and return a more generic error."
-  spec.homepage      = "http://github.com/appfolio/ae_network_connection_exception"
-  spec.license       = "MIT"
-
-  spec.files         = `git ls-files -z`.split("\x0")
-  spec.require_paths = ["lib"]
+  spec.author        = 'Appfolio'
+  spec.email         = 'dev@appfolio.com'
+  spec.summary       = 'Provides sane exceptions for network failures'
+  spec.description   = 'Catch exceptions related to establishing a network connection and return a generic error.'
+  spec.homepage      = 'http://github.com/appfolio/ae_network_connection_exception'
+  spec.license       = 'MIT'
+  spec.files         = Dir['**/*'].select { |f| f[%r{^(lib/|Gemfile$|Rakefile|README.md|LICENSE.txt|.*gemspec)}] }
+  spec.require_paths = ['lib']
 
-  spec.add_development_dependency "bundler", "~> 1.7"
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency('appraisal', '~> 2.2')
+  spec.add_development_dependency('bundler', ['>= 1.7', '< 3.0'])
+  spec.add_development_dependency('minitest', '~> 5.0')
+  spec.add_development_dependency('minitest-reporters', '~> 1.4')
+  spec.add_development_dependency('rake', '~> 10.0')
+  spec.add_development_dependency('rest-client', '~> 2.0')
+  spec.add_development_dependency('rubocop', '~> 0.80')
+  spec.add_development_dependency('rubocop-minitest', '~> 0.6.2')
+  spec.add_development_dependency('simplecov', '~> 0.17')
```

Comment:
Can you do `~> 0.18`?

Reply:
I tried to do `~> 0.18` but it requires Ruby `2.4`. If we still want to support Ruby `2.3.3` then we have to keep this at the lower version.

Reply:
I forgot about that. Yeah I guess let's stay on the older version.